client/starwhale/base/context.py

```diff
@@ -21,9 +22,12 @@ def __init__(
         index: int = 0,
         dataset_uris: t.Optional[t.List[str]] = None,
         version: str = "",
-        project: str = "",
+        run_project: t.Optional[Project] = None,
+        log_project: t.Optional[Project] = None,
     ):
-        self.project = project
+        self.run_project = run_project or Project()
+        self.project = self.run_project.name
```

Comment:
can we replace `self.project` with `self.run_project`?

Reply:
done